### PR TITLE
First pass of exit code implementation.

### PR DESCRIPTION
**Pull request by fubarhouse**
This is a first pass at getting exit codes implemented which allows for future compatibility for expansion of this feature if ever considered.

It provides a new variable available to the `Assessment` with a getter and setter, and provides one condition where the default value `0` is changed to `1`, so the code will always execute and complete with exit code `0` unless configured to use a different exit code.

Feedback is appreciated, hopefully it lines up well enough with the discussion in #211.

### CODE REVIEW

File: src/Audit/Drupal/ModuleEnabled.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ public function remediate(Sandbox $sandbox)
   {
     $module = $sandbox->getParameter('module');
     $sandbox->drush()->en($module, '-y');
-    return $this->check($sandbox);
+    return $this->audit($sandbox);
```
Comment (fiasco) [inline src/Audit/Drupal/ModuleEnabled.php]: This one should be done in a seperate PR. Can we remove it from this PR?

**Comment by fiasco** [comment]
I've added onto your fork to head more towards the direction I was thinking. Could you test it out?

**Comment by fubarhouse** [comment]
I've just tested it and I'm happy.
Looks good and helps guide direction.

File: src/Audit/Drupal/ModuleEnabled.php (hunk 1 above)
Comment (fubarhouse) [reply src/Audit/Drupal/ModuleEnabled.php]: This was already merged, so I'll remove it.

**Comment by fubarhouse** [comment]
I've remediated from your code review - let me know if can help this along.

**Comment by fiasco** [comment]
Thanks @fubarhouse !

**Pull request merged**